.github/workflows/ci.yaml

```
@@ -151,7 +151,7 @@ jobs:
   prepare-tests-linux:
     name: tests / prepare / ${{ matrix.python-version }} / Linux
     runs-on: ubuntu-latest
-    timeout-minutes: 15
+    timeout-minutes: 5
```

```suggestion
    timeout-minutes: 6
```

In order to relaunch the tests.